paper

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env bash
+
+# Can't use `set -eu` since ppl might source this file
+
+# TODO: Display warning that one should use gradle?
```

Comment:
Wrap it in a subshell, then. It's stupid, but whatever.

Reply:
I'm not sure how it applies here? This is my second point from things to consider in the PR description:
> 2. Should (most) commands warn that one should use gradle commands now, instead?

Do we want to let PPL know that gradle replaced (most of) this script and they can use it directly for most things? Probably with an env variable to suppress it.

Reply:
Oh, I feel like your comment might have been to that third line - can't do that, since we can't use subshells for the whole scripts due to us wanting to be able to change user CWD after some of the commands.